### ISSUE TITLE
opneid4vp metadata including redirect_uris and request_uris sanification and presentation_definition

### ISSUE THREAD

**Issue by peppelinux** [peppelinux/federation-wallet#11]
The current draft has introduced at least three brand new metadata parameters about openid4vp implementations.
These was introduced for the first time in the following openid4vp github issue:

https://github.com/openid/OpenID4VP/issues/17#issuecomment-1702545425

**Comment by jogu** [comment]
I think an issue in the OID4VP github is needed for each new proposed metadata and as I understand it in the case of Italy the proposal is not to use the new parameters in the `client_metadata` request parameter anyhow and hence https://github.com/openid/OpenID4VP/issues/17 (which is about the `client_metadata` request parameter) isn't too relevant?

**Comment by selfissued** [comment]
I agree that we should have separate tracking issues for each proposed new metadata parameter.  Then they can each be easily considered on their merits.
